apache/zeppelin
# [ZEPPELIN-2492] Use single Dockerfile for each tag

### What is this PR for?

Use single `Dockerfile` for each release since [apache infra uses tag pushes](https://issues.apache.org/jira/browse/INFRA-12781) to build an image. 

- https://issues.apache.org/jira/browse/INFRA-12781

After release process finishes, dockerhub will build using the pushed tag.

### What type of PR is it?
[Improvement]

### What is the Jira issue?

[ZEPPELIN-2492](https://issues.apache.org/jira/browse/ZEPPELIN-2492)

### How should this be tested?

1. `./dev/change_zeppelin_version.sh 0.8.0-SNAPSHOT 0.7.1`
2. Check that version is properly set: `vi scrtips/docker/zeppelin/bin/Dockerfile`
3. Build docker image `cd scripts/docker/zeppelin/bin; docker build -t zeppelin:0.7.1 ./`
4. Run the image: `docker run -p 8080:8080  --rm --name zeppelin zeppelin:0.7.1`

### Screenshots (if appropriate)

NONE

### Questions:
* Does the licenses files need update? - NO
* Is there breaking changes for older versions? - NO
* Does this needs documentation? - NO
